build-tools-internal/version.properties

```diff
@@ -42,6 +42,12 @@ junit5            = 5.7.1
 hamcrest          = 2.1
 mocksocket        = 1.2
 
+# test container dependencies
+testcontainer     = 1.19.2
+docker            = 3.3.4
```

Comment:
nit: Can we rename this `docker-java` since that's actually what it is?

Reply:
fixed. Needed to add 

```    
T systemProperty(String setting, Supplier<String> value, Predicate<LocalNodeSpec> predicate);
``` for this

Reply:
fixed